pinc/access_log.inc

```diff
@@ -1,22 +1,19 @@
 <?php
+include_once($relPath.'misc.inc');
```

Comment:
Bringing this in has no unintended consequences (performance? side effects?)

Reply:
So, the only side effects should be to bring a bunch of utility functions (and 2 empty Exception classes) into the global namespace.

If I understand things correctly, there shouldn't really be any noticeable performance issues (apart from possibly an extra `stat()` syscall to check `misc.inc` hasn't changed) because every `.php` file will end up being precompiled and stored in the opcode cache.

In any event, all 3 files that include `access_log.inc` already include `misc.inc`.

Reply:
<nod> This is very likely a no-op since other things include `misc.inc`.